File: Lista 2 sala/C/atividade08.c
#include <stdio.h>

int main() {
    int a, b, c;
    int algum_divisivel = 0;

    printf("Diga-me um número X: ");
    scanf("%d", &a);

    printf("Diga-me um número Y: ");
    scanf("%d", &b);

    printf("Diga-me um número Z: ");
    scanf("%d", &c);

    if (a % 2 == 0 && a % 3 == 0) {
        printf("É divisível %d\n", a);
        algum_divisivel = 1;
    }

    if (b % 2 == 0 && b % 3 == 0) {
        printf("É divisível %d\n", b);
        algum_divisivel = 1;
    }

    if (c % 2 == 0 && c % 3 == 0) {
        printf("É divisível %d\n", c);
        algum_divisivel = 1;
    }

    if (!algum_divisivel) {
        printf("Nenhum número é divisível por 2 e 3.\n");
    }

    return 0;
}

File: Lista 2 sala/Python/atividade08.py
# 8)- Ler 3 números inteiros e exibir os números que são divisíveis por 2 e 3.

a = int(input("Diga-me um número X: "))
b = int(input("Diga-me um número Y: "))
c = int(input("Diga-me um número Z: "))

algum_divisivel = False

if a % 2 == 0 and a % 3 == 0:
    print("É divisivel",a)
    algum_divisivel = True

if b % 2 == 0 and b % 3 == 0:
    print("É divisivel",b)
    algum_divisivel = True

if c % 2 == 0 and c % 3 == 0:
    print("É divisivel",c)
    algum_divisivel = True

if not algum_divisivel:
    print("Nenhum número é divisível por 2 e 3.")
